source/base/function_parser.cc

```diff
@@ -48,7 +48,7 @@ template <int dim>
 FunctionParser<dim>::FunctionParser(const unsigned int n_components,
                                     const double       initial_time)
   :
-  Function<dim>(n_components, initial_time)
+  AutoDerivativeFunction<dim>(1e-8, n_components, initial_time)
```

Comment:
Hm, is 1e-8 always appropriate? What if someone does general relativity in SI units? I think it may be better to make this an argument to the constructor of the class (maybe with default value of 1e-8) and then pass this value to the base class.